keyboards/afternoonlabs/breeze/keymaps/southpaw/keymap.c

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2021 eithanshavit
```

Comment:
Can this be included within the default keymap instead? So the default keymap for the southern breeze layout is this?